app/interactors/fixed_streams/stream_daily_averages_interactor.rb

```diff
@@ -0,0 +1,23 @@
+module FixedStreams
+  class StreamDailyAveragesInteractor
```

Comment:
When it comes to naming, I would name a module the same as its corresponding controller, and a class after the action it refers to.

```
module StreamDailyAverages
  class IndexInteractor
```